vizro-core/src/vizro/models/_navigation/nav_link.py

```diff
@@ -57,10 +58,14 @@ def build(self, *, active_page_id=None):
 
         button = dbc.Button(
             [
-                html.Span(self.icon, className="material-symbols-outlined"),
-                # TODO: commented out until we insert styling for the tooltip or find a better way to display it (e.g.
-                # try dbc.Popover or Dash mantine components tooltip?).
-                # dbc.Tooltip(html.P(self.label), target=self.id, placement="bottom", className="custom-tooltip"),
+                dmc.Tooltip(
+                    label=self.label,
+                    offset=6,
+                    withArrow=True,
+                    zIndex=1,
+                    children=[html.Span(self.icon, className="material-symbols-outlined")],
```

Comment:
In the designs there is actually a max. width for the tooltip defined, so in case it exceeds it, text wrapping will be activated automatically. We should have this as our default as well, but optimally users should be able to overwrite it via CSS if they want.

See Figma designs for details:
<img width="249" alt="Screenshot 2023-12-05 at 13 30 43" src="https://github.com/mckinsey/vizro/assets/90609403/3e8a5232-93a2-4aaa-9d7c-d8c57874b2a1">